Makefile

```diff
@@ -11,3 +11,8 @@ test:
 	pip install -r $(SRC)/requirements.txt
 	pip install pytest
 	python -m pytest tests/
+
+devtest:
+	pip install -r $(SRC)/requirements.txt
+	pip install pytest
+	python -m pytest tests/ -m dev
```

Comment:
This line errors oin my machine with sth along the lines:

```
python -m pytest -m dev tests/ 
============================= test session starts ==============================
platform linux -- Python 3.8.10, pytest-7.1.2, pluggy-1.0.0
rootdir: /home/mjansen/code/actinia-python-client
collected 14 items / 14 deselected / 0 selected                                

============================ 14 deselected in 0.13s ============================
make: *** [Makefile:18: devtest] Fehler 5
```

Reply:
I will add a description how you can use it

Reply:
@marcjansen please check if it is now ok

Reply:
Great, now I understand that this expected behaviour. Thanks for the clarification